filebeat/docs/modules/awsfargate.asciidoc

```diff
@@ -0,0 +1,139 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-awsfargate]]
+[role="xpack"]
+
+:libbeat-xpack-dir: ../../../x-pack/libbeat
+
+:modulename: awsfargate
+:has-dashboards: false
+
+== AWS Fargate module
+
+beta[]
+
+This is a module for collect container logs from Amazon ECS on Fargate.
```

Comment:
```suggestion
This module can be used to collect container logs from Amazon ECS on Fargate.
```

There is another place to apply this change (in case you agree) too, so please don't rely just on this GH `commit`.

Reply:
Will do :) Thanks!